Demo/Sources/Model/ServerSetting.swift

```diff
@@ -12,6 +12,19 @@ enum ServerSetting: Int, CaseIterable {
     case production
     case stage
     case test
+
+    case productionCH
+    case stageCH
+    case testCH
+
+    case productionWW
+    case stageWW
+    case testWW
+
+    case samProduction
+    case samStage
+    case samTest
```

Comment:
As we prefix these cases with `sam`, should we also prefix the original cases with `il`?

Reply:
Done.